.github/workflows/pathogen-repo-ci.yaml

```diff
@@ -237,14 +246,17 @@ jobs:
           fi
 
       - run: nextstrain build . ${{ inputs.build-args }}
+        working-directory: ${{ inputs.workflow-root }}
 
       - if: always()
         uses: actions/upload-artifact@v3
+        env:
+          workdir: ${{ inputs.workflow-root }}
         with:
           name: ${{ inputs.artifact-name }}-${{ matrix.runtime }}
           path: |
-            auspice/
-            results/
-            benchmarks/
-            logs/
-            .snakemake/log/
+            $workdir/auspice/
+            $workdir/results/
+            $workdir/benchmarks/
+            $workdir/logs/
+            $workdir/.snakemake/log/
```

Comment:
This currently only accepts paths without a trailing slash. I think it'd be good to accept both with and without, because even if we document support for only one in the input description, the error that happens here might not be very informative and only happens after everything else is done running.

I don't think there's an easy way to do something like Python's `os.path.join` inline here, so the solution might be to normalize as a first step before using the value.

Reply:
This sounds like a non-blocking feature request to me :)

Reply:
Yeah it works, so this can be non-blocking.

Reply:
> This sounds like a non-blocking feature request to me :)

This sounds like a flippant response to a reasonable request for robustness.

The trailing slash is not even a documented requirement in the input description. But even if it was, requiring a slash is an easily-avoided tripping hazard that we don't need to subject ourselves to.